packages/react/src/Popover/Popover.module.css

```diff
@@ -205,4 +205,75 @@
       bottom: calc(var(--base-size-16) + 1px);
     }
   }
+
+  &:where([data-width-small]) {
+    width: 256px;
```

Comment:
I think we should update all of the widths to use the design tokens: https://primer.style/product/primitives/size/#overlay